apps/custom-config/g-sensor.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+var property = require('@yoda/property')
+var BaseConfig = require('./base-config')
+var logger = require('logger')('custom-config-g-sensor')
+
+/**
+ * g-sensor handler
```

Comment:
@extends BaseConfig is missing

Reply:
What's your meaning? The file `base-config.js` exists.

Reply:
You comments are incomplete to generate the API docs.

```js
/**
 * @class Gsensor
 * @extends EventEmitter
 */
```

Reply:
OK, updated.